.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedenc
+* @tkxkd0159 @zemyblue @0Tech @jaeseung-bae
```

Comment:
CRLF

Reply:
I copy and paste this file. https://github.com/Finschia/finschia-sdk/blob/main/.github/CODEOWNERS I think it's no problem.

Reply:
OK.